modules/mpv-player/android/src/main/java/expo/modules/mpvplayer/nativeplayer/ui/PlayerTopBar.kt

```
@@ -0,0 +1,655 @@
+            }
+        }
+    }
+}
+
+fun formatTimeSec(seconds: Double): String {
+    val total = seconds.toInt().coerceAtLeast(0)
+    val hrs = total / 3600
+    val mins = (total % 3600) / 60
+    val secs = total % 60
+    return if (hrs > 0) {
+        String.format("%d:%02d:%02d", hrs, mins, secs)
```

detekt (1.23.8)

[warning] String.format("%d:%02d:%02d", hrs, mins, secs) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)